src/Modules/UngradedModules.jl

```diff
@@ -8712,3 +8716,258 @@ function dual(f::ModuleFPHom{<:ModuleFP, <:ModuleFP, Nothing}; # Third parameter
               for phi in gens(N_dual)])
 end
 
+##########################################################################
+## Functionality for modules happening to be finite dimensional vector
+## spaces
+##########################################################################
+@doc raw"""
+    vector_space_dimension(M::SubquoModule, d::Int)
+
+Let R be a MPolyAnyRing over a field kk and let M be a subquotient module over R.
+Then the command returns the dimension of the kk-vectorspace corresponding to the
+degree d slice of M, where the degree of each variable of R is counted as one and
+the one of each generator of the ambient free module of M as zero.
+
+    vector_space_dimension(M::SubquoModule)
+
+If M happens to be finite-dimensional as a kk-vectorspace, this returns its dimension.
```

Comment:
```suggestion
Let ``R`` be a `MPolyAnyRing` over a field ``k`` and let ``M`` be a subquotient module over ``R``.
Then the command returns the dimension of the ``k``-vectorspace corresponding to the
degree ``d`` slice of ``M``, where the degree of each variable of ``R`` is counted as one and
the one of each generator of the ambient free module of ``M`` as zero.

    vector_space_dimension(M::SubquoModule)

If ``M`` happens to be finite-dimensional as a ``k``-vectorspace, this returns its dimension.
```
and what if it is infinite dimensional?